test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -581,3 +603,12 @@ test.requestHooks(mockSelectAuthenticatorForRecovery)('should not show custom fa
 
   await t.expect(await pageObject.factorPageHelpLinksExists()).notOk();
 });
+
+test.requestHooks(mockChallengeCustomApp)('should navigate to Custom App challenge page', async t => {
+  const selectFactorPage = await setup(t);
+  await t.expect(selectFactorPage.getIdentifier()).eql('testUser@okta.com');
+  await t.expect(selectFactorPage.getFormTitle()).eql('Verify it\'s you with a security method');
+  selectFactorPage.selectFactorByIndex(9);
+  const challengeFactorPage = new ChallengeFactorPageObject(t);
+  await t.expect(challengeFactorPage.getFormTitle()).eql('Verify with Custom Push');
```

Comment:
nit: may be also check for the presence of custom app logo

Reply:
app logo and other details in the page are already verified [here](https://github.com/okta/okta-signin-widget/blob/33291ef94d29bbc9543a0b3fe8b8aa36f1549b54/test/testcafe/spec/ChallengeCustomAppPush_spec.js#L81). The scope of this test is just to ensure the page navigates correctly.